### DISCUSSION TITLE
Will this plugin work with FS2024?

### DISCUSSION THREAD

**Discussion by Getreed** [calibx/msfsdeck#258] [Q&A]
Hey Calibx, 

I recently purchased the Loupedeck Live to use with Flight Simulator 2024. Will your plugin be compatible with 2024, or do you have plans to create one for that version? I am truly impressed by the plugin you have developed and hope to see it available for 2024 as well.  Either way, thank you for sharing your creation with the rest of the world!

**Comment by Znorr** [comment]
having this again for MSFS 2024 would be just awesome!
i am so used to the Loupedeck, that i dont want to miss it.

Thanks for your effort!!

**Comment by calibx** [comment]
Hi, if you use LoupeDeck 5.9, the marketplace release of the plugin and if you disable automatic profile switch in the LoupeDeckUI : nearly all inputs should works.
For the 6.0 : the new release is in alpha test, I will try to include the MSFS 2024 profile too

**Reply by Znorr** [reply]
thanks! working fine with the 5.9 version.

**Comment by Getreed** [comment]
Thank you for the quick response!  That’s great to hear! I will be stepping down to 5.9 in order to utilize your plugin.  While I do have FCPX, the sim is the main reason I have purchased the Loupedeck. Again, I really appreciate your efforts and allowing us to utilize your plugin.  On Nov 23, 2024, at 08:54, calibx ***@***.***> wrote:﻿
Hi, if you use LoupeDeck 5.9, the marketplace release of the plugin and if you disable automatic profile switch in the LoupeDeckUI : nearly all inputs should works.
For the 6.0 : the new release is in alpha test, I will try to include the MSFS 2024 profile too

**Comment by calibx** [comment]
Now it works :p
